CCDScan release August 2023 (#868)
Concordium/concordium.github.io: source/mainnet/net/release-notes/release-notes-lp.rst

CCDScan
-------

>     August 21, 2023

Reviewer: ```suggestion
    August 16, 2023
```

>         Version X.X.X contains support for `protocol version 6 <https://github.com/Concordium/concordium-update-proposals/blob/main/updates/P6.txt>`_ with Concordium BFT consensus which will be released August 21, 2023.

Reviewer: ```suggestion
        Version CCD frontend 1.4.0 and backend 1.6.2 contains support for `protocol version 6 <https://github.com/Concordium/concordium-update-proposals/blob/main/updates/P6.txt>`_ with Concordium BFT consensus which will be released August 21, 2023.
```

>         Version CCD frontend 1.4.0 and backend 1.6.2 contains support for `protocol version 6 <https://github.com/Concordium/concordium-update-proposals/blob/main/updates/P6.txt>`_ with Concordium BFT consensus which will be released August 21, 2023.

Reviewer: ```suggestion
        Version CCD frontend 1.4 and backend 1.6 contains support for `protocol version 6 <https://github.com/Concordium/concordium-update-proposals/blob/main/updates/P6.txt>`_ with Concordium BFT consensus which will be released August 21, 2023.
```
